wordle-web/src/scripts/wordleGame.ts

```diff
@@ -71,9 +78,45 @@ export class WordleGame {
     }
   }
 
+  setPlayerName(name: string) {
+    if (name != '' || name != null) {
+      this.currentPlayer = name
+    } else {
+      this.currentPlayer = 'Guest'
+    }
+  }
+
   removeGuess() {
     while (this.guess.text !== '') {
       this.guess.pop()
     }
   }
+
+  postPlayerToApi(name: string, attempts: number) {
+    Axios.post('https://wordlewebapp2023.azurewebsites.net/Player', {
+      playerName: name,
+      gameCount: 1,
```

Comment:
Your API doesn't seem to use gameCount and, with it always being 1, I think you can safely remove it from your Axios call and your API.